src/main/java/dbProcs/Database.java

```diff
@@ -164,7 +172,7 @@ public static Connection getSqlInjLessonConnection(String ApplicationRoot)
 	   try
 	   {
 		   //Pull Driver and DB URL out of database.properties
-		   String props = ApplicationRoot+"/WEB-INF/classes/database.properties";

```

Comment:
Removing the applications running context from each db call has not broken the web app as System.getProperty("catalina.base") in tomcat returns the same thing. However changing this has broken all of the JUnit tests as System.getProperty("catalina.base") returns null.

I guess the fact the we don't have the property files in the workspace anymore also contributes to this. We need a solution to this so that the JUnit tests can run before this can be merged